api/customer.go

```diff
@@ -209,3 +209,35 @@ func (c Call) GetCustomerBalances(ctx context.Context, customerID uuid.UUID) (mo
 	fL.Info().Interface(model.LogStrResponse, response.Data).Msg("response")
 	return response.Data, nil
 }
+
+func (c Call) DeleteCustomer(ctx context.Context, customerID uuid.UUID) error {
+	endpoint := fmt.Sprintf("%s%s/%s", c.baseURL, customerAPIVersion, customerID)
+
+	fL := c.logger.With().Str("func", "DeleteCustomer").Str("endpoint", endpoint).Logger()
+	fL.Info().Msg("starting...")
+	fL.Info().Str("customerID", customerID.String()).Interface(model.LogStrRequest, "empty").Msg("request")
+
+	defer fL.Info().Msg("done...")
+
+	var response interface{}
+
+	res, err := c.client.R().
+		SetAuthToken(c.bearerToken).
+		SetResult(&response).
+		SetContext(ctx).
+		Delete(endpoint)
+
```

Comment:
`c.client.R()` is an instance of what please

Reply:
`client       *resty.Client`